data/35e/wizards_of_the_coast/supplement/book_of_exalted_deeds/be_abilities.lst

```diff
@@ -287,7 +287,7 @@ Fey Transformation		KEY:Swanmay ~ Fey Transformation					CATEGORY:Special Abilit
 
 # Sword of Righteousness
 Weapon and Armor Proficiency	KEY:Sword of Righteousness ~ Weapon and Armor Proficiency	CATEGORY:Special Ability	TYPE:Class Feature.Sword of Righteousness Class Feature.SpecialQuality			DESC:Swords of righteousness gain no proficiency with any weapon or armor.																																																																																																																																																																																																																																																																																																																		SOURCEPAGE:p.78
-Bonus Exalted Feat		KEY:Sword of Righteousness ~ Bonus Exalted Feat			CATEGORY:Special Ability	TYPE:Class Feature.Sword of Righteousness Class Feature.SpecialQuality			DESC:At every level, a sword of righteousness gets a bonus exalted feat. These bonus feats must be drawn from among those identified as exalted feats in Chapter 4: Feats.																																																																																																																																																																																																																																																																																																		SOURCEPAGE:p.78
+Bonus Exalted Feat		KEY:Sword of Righteousness ~ Bonus Exalted Feat			CATEGORY:Special Ability	TYPE:Class Feature.Sword of Righteousness Class Feature.SpecialQuality			DESC:At every level, a sword of righteousness gets a bonus exalted feat. These bonus feats must be drawn from among those identified as exalted feats in Chapter 4: Feats.							DEFINE:Pool_ExaltedFeat|0		BONUS:VAR|Pool_ExaltedFeat|SwordofRighteousnessBonusExaltedFeatCount																																																																																																																																																																																																																																																																																										SOURCEPAGE:p.78
```

Comment:
Looks good, only technique change is remove a "var-bonuses-var" triple chain.
Instead of the count, the more efficient solution is BONUS:VAR|Pool_x|(ClassNameLVL>=x) method for level increment.